benefits/core/models.py

```diff
@@ -206,21 +206,12 @@ class PaymentProcessor(models.Model):
     id = models.AutoField(primary_key=True)
     name = models.TextField()
     api_base_url = models.TextField()
-    api_access_token_endpoint = models.TextField()
-    api_access_token_request_key = models.TextField()
-    api_access_token_request_val = models.TextField()
+    client_id = models.TextField()
+    client_secret_name = SecretNameField()
+    audience = models.TextField()
     card_tokenize_url = models.TextField()
     card_tokenize_func = models.TextField()
     card_tokenize_env = models.TextField()
-    # The certificate used for client certificate authentication to the API
-    client_cert = models.ForeignKey(PemData, related_name="+", on_delete=models.PROTECT)
-    # The private key, used to sign the certificate
-    client_cert_private_key = models.ForeignKey(PemData, related_name="+", on_delete=models.PROTECT)
-    # The root CA bundle, used to verify the server.
-    client_cert_root_ca = models.ForeignKey(PemData, related_name="+", on_delete=models.PROTECT)
-    customer_endpoint = models.TextField()
-    customers_endpoint = models.TextField()
-    group_endpoint = models.TextField()
 
```

Comment:
Model refactor looks good, to store the name of the `client_secret` :+1: 

But we will still need the _value_ referenced by the `client_secret_name` for later passing into the `littlepay.api.Client`.

Suggest creating a `@property` like `client_secret` that uses our helper `get_secret_by_name()` -- see e.g. [`AuthProvider.client_id`](https://github.com/cal-itp/benefits/blob/dev/benefits/core/models.py#L102) for an example.

Reply:
Thanks for catching that 🙏  

I added the [`@property`](https://github.com/cal-itp/benefits/pull/1905/commits/6e238c3bd24e2836770cb44b59c0ff5c8096f592#diff-4d37ea08e0da7b3a7ea86f5fa29f0a86a74acb8089d2c9bd881173c430fad818R228-R230) in [6e238c3](https://github.com/cal-itp/benefits/pull/1905/commits/6e238c3bd24e2836770cb44b59c0ff5c8096f592)